src/Base64.js

```diff
@@ -0,0 +1,13 @@
+const base64 = require("base64-js");
+
+exports.fromByteArray = base64.fromByteArray;
+
+exports.toByteArray = base64.toByteArray;
+
+exports._decodeBase64 = maybe => str => {
+  try {
+    return maybe.just(base64.toByteArray(str));
+  } catch (_) {
+    return maybe.nothing;
+  }
+};
```

Comment:
I don't think we need to add a new library for this - we already have a shim for nodes buffer set up in webpack to allow browsers to use it. We can use `node-buffer` which is already a dependency and https://pursuit.purescript.org/packages/purescript-node-buffer/8.0.0/docs/Node.Encoding#v:Base64

Reply:
I don't get how to use that package. Is it only interfaces? There isn't a single instance of [MutableBuffer](https://pursuit.purescript.org/packages/purescript-node-buffer/8.0.0/docs/Node.Buffer.Class#t:MutableBuffer) class.

Reply:
base64-js is 150 lines of pure JS, it's not a big deal. As a bonus it would be easier to switch away from webpack.

Reply:
Well, we don't have to use that purescript library, but IMO it's still better to use what we already have. We're already doing something very similar over in `TypedArray`, so we'd at least be being consistent:
https://github.com/Plutonomicon/cardano-transaction-lib/blob/6c018d43ddfef771bfd586e885ecb0cc7ebd8421/src/Types/ByteArray.js#L33

I think webpack only saves us from explicitly requiring buffer, we could move away from webpack and still keep the shim.

...

Just had a look into it, the `Buffer` shim's b64 conversion is implemented using `base64-js` anyway. I'm still slightly inclined to prefer using one interface for conversions but I don't think it matters, up to you

Reply:
When using Buffer directly, we lose validity checks for base64.

```
> new Uint8Array(Buffer.from('#$#$!(*#)(*$#)*', 'base64'))
Uint8Array(0) []
```

Reply:
> I'm still slightly inclined to prefer using one interface for conversions

Also note that we don't use `MutableBuffer` interface elsewhere. We only depend on `node-buffer` to pass an `Encoding` value to `readTextFile` in one place.